pkg/ui/react-app/src/pages/graph/PanelList.tsx

```diff
@@ -204,6 +213,14 @@ const PanelList: FC<RouteComponentProps & PathPrefixProps> = ({ pathPrefix = ''
           </Checkbox>
         </div>
         <div className="float-right">
+          <Dropdown
```

Comment:
Maybe having this select item in `Panel` instead of `PanelList` would be a better idea, similar to the `Use Deduplication` option?
The reason for this is that we might want to compare the results of two engines on the same screen, with different query panels. So I think having the dropdown on every query panel would be amazing.